heron/spi/src/java/com/twitter/heron/spi/common/keys.yaml

```diff
@@ -62,6 +62,12 @@ SCHEDULER_IS_SERVICE:                heron.scheduler.is.service
 SCHEDULER_JAR:                       heron.jars.scheduler
 
 ################################################################################
+# keys for Topology Update command
+################################################################################
+TOPOLOGY_UPDATE_DEACTIVATE_SECONDS:  heron.topology.update.deactivate.seconds
+TOPOLOGY_UPDATE_REACTIVATE_SECONDS:  heron.topology.update.reactivate.seconds
```

Comment:
@billonahill I am not thinking of a network callback. Say a new interface `UpdateCallback` is added. A new `scheduler` config could be added using which the callback handler could be configured. The default implementation could be no op handler. User could configure a provided time based callback handler or a custom handler. This would not need a network call or protocol implementation.